src/nft/utils/listNfts.ts

```diff
@@ -147,21 +145,20 @@ export async function signListing(
                 itemType: ItemType.ERC721,
                 token: asset.asset_contract.address,
                 identifier: asset.tokenId,
-                amount: '1',
               },
             ],
             consideration: considerationItems,
             endTime: asset.expirationTime.toString(),
-            zone: OPENSEA_DEFAULT_ZONE,
-            restrictedByZone: true,
+            zone: ZERO_ADDRESS,
             allowPartialFills: true,
           },
           signerAddress
         )
 
         const order = await executeAllActions()
+        const seaportV14Order = { ...order, protocol_address: OPENSEA_SEAPORT_V1_4_CONTRACT }
```

Comment:
why not just change the order response to include this since this is going to be the new contract going forward?

Reply:
executeAllActions is from seaport.createOrder so I can't change the response type, I just need to include this `protocol_address` param in the body of the api request. I tried adding it here and modifying the payload in `PostOpenSeaSellOrder` and this was cleaner imo